app/javascript/src/Plans/components/ApplicationPlansTableCard.jsx

```diff
@@ -0,0 +1,122 @@
+// @flow
+
+import * as React from 'react'
+
+import { Card } from '@patternfly/react-core'
+// $FlowIgnore[missing-export] export is there, name_mapper is the problem
+import { ApplicationPlansTable } from 'Plans'
+import * as alert from 'utilities/alert'
+import { post, ajax } from 'utilities/ajax'
+import { safeFromJsonString } from 'utilities/json-utils'
+import { confirm } from 'utilities/confirm-dialog'
+
+import type { ApplicationPlan, Action } from 'Types'
+
+export type Props = {
+  plans: ApplicationPlan[],
+  count: number,
+  searchHref: string,
+}
+
+const ApplicationPlansTableCard = ({ plans: initialPlans, count, searchHref }: Props): React.Node => {
+  const [plans, setPlans] = React.useState<ApplicationPlan[]>(initialPlans)
+  const [isLoading, setIsLoading] = React.useState<boolean>(false)
+
+  const handleActionCopy = (path) => post(path)
+    .then(data => data.json()
+      .then(res => {
+        if (data.status === 201) {
+          alert.notice(res.notice)
```

Comment:
Maybe unrelated to this PR (ignore if convenient), but calling this `alert` is a bit confusing, maybe we should call it `flash` or `flashMessage` instead.

Reply:
Yes, from a backend point of view this might seem confusing. However, we are transitioning to Patternfly and I think we should get used to Alerts (https://www.patternfly.org/v4/components/alert).
Anyway, our "alert" module was introduced recently and it's still very young, we can changed it however we see fit, although, not in this PR 😄